examples/mdb-rar/src/main/java/org/wildfly/plugins/demo/mdb/ActiveMQMDB.java

```diff
@@ -0,0 +1,33 @@
+package org.wildfly.plugins.demo.mdb;
```

Comment:
This is just a question, but should we add a copyright header here? I don't think it matters overall, I'm just curious.

Reply:
None of our examples have copyrights, I am not sure of the policy for examples.

Reply:
AIUI there is no strict policy. Since none of them have it I think it's fine. My personal preference is just to make it consistent with either having them or not having them :)